packages/base/any/kernels/5.10-lts/patches/0043-switchdev-prestera-v3.1.1.patch

```diff
@@ -0,0 +1,11856 @@
+From 003f41f494d24763d8dfee958931c968e12397ea Mon Sep 17 00:00:00 2001
+From: Taras Chornyi <taras.chornyi@plvision.eu>
+Date: Fri, 17 Dec 2021 14:07:28 +0200
+Subject: [PATCH] prestera: Update driver to v3.1.1
+
+Switchdev Driver v3.1.1 includes:
+
+ - Supporting Marvell 98DX3500-A1 Packet Processor (AC5X)
+ - Compatible with all current known platforms designed based Marvell
+   Switchdev HW Design Guidelines.
+ - Bug fixing
+
+Tested by Marvell Validation team on all based DNI/Accton platforms
+
+Signed-off-by: Taras Chornyi <taras.chornyi@plvision.eu>
```

Comment:
Who reviewed it?

Reply:
It was reviewed internally by a team members